src/components/graph-view.js

```diff
@@ -500,12 +503,18 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
     );
 
     // inform consumer
-    this.props.onSelectNode(null);
-    this.props.onDeleteNode(selectedNode, nodeId, newNodesArr);
+    if (onSelectNode) {
+      onSelectNode(null);
```

Comment:
one option could be to default prop to () => {} so it means you don't need if conditions throughout the code?

Reply:
That's a good idea. I'm planning a rewrite using functions so I will consider it in the next iteration.